### DISCUSSION TITLE
how to get 0.25 per block won?

### DISCUSSION THREAD

**Discussion by AleksandrPDA** [Chia-Network/chia-blockchain#8649] [Support]
Please help me figure out how I can get a reward for the block I won. My payment address to the farmer: xch1689rwj9vtr7dwzmsh6d4qzenjl564cd8vfq59ayp484psxtfvdzqka8lhm,
Block # 897984 
Hash 0eff69e05adf64fd0ca1438807bc082ea251e7ad2950aa793718daedf3c9c3b4
Thank you very much in advance !!!

**Comment by emlowe** [comment]
If at any time you were with corepool, Hpool, or poolharvest.io (OG only), and downloaded or used their software your farmer rewards are likely set up for those pools in config.yaml and need to be changed.
Block # 897984 , farmer rewards went to: `xch14uwnt8rpthz6acrraveylp4sus375t5ts8hh9hg8j2rth66ravkqw2vgf4`
The plot public key is `0xa30b8e565e0ad1b2d14065b449a82bf450f4e501290663a614516882b8ee536a93595219d2b38d3a3cf73ee05d8996f1` - you can check this is your plot from the GUI, but it's not very easy

Space pool has a good blog post on this topic: https://blog.pool.space/how-to-find-your-0-25xch-after-winning-a-block-in-the-pool-fc61d0bee5c5

**Reply by AleksandrPDA** [reply]
Thank you for your help! I've never been with corepool and Hpool. only worked with space pool. 
I should try to change the address

**Reply by AleksandrPDA** [reply]
I read this topic in the block:  https://blog.pool.space/how-to-find-your-0-25xch-after-winning-a-block-in-the-pool-fc61d0bee5c5 , I checked everything as they say there and concluded that I had a correct address and why then I did not receive a reward is not clear

**Comment by plot14** [comment]
I have the same problem, I think we have all been defrauded. and there is no pre and post execution assistance for the chia project
